model/atmosphere/dycore/src/icon4py/model/atmosphere/dycore/nh_solve/solve_nonhydro.py

```diff
@@ -411,9 +438,12 @@ def __init__(self, config: NonHydrostaticConfig):
 
 class SolveNonhydro:
     def __init__(
-        self, exchange: decomposition.ExchangeRuntime = decomposition.SingleNodeExchange()
+        self,
+        exchange: decomposition.ExchangeRuntime = decomposition.SingleNodeExchange(),
+        backend: backend.Backend = None,
```

Comment:
If I remember correctly, for the `Diffusion` we opted to not add a default. Can you do the same here?

Reply:
yes sorry, I missed this